priv/perf/config/config.exs

```diff
@@ -9,15 +9,17 @@ ethereum_client_timeout_ms = 20_000
 
 config :ethereumex,
   http_options: [recv_timeout: ethereum_client_timeout_ms],
-  url: System.get_env("ETHEREUM_RPC_URL")
+  url: System.get_env("ETHEREUM_RPC_URL") || "http://localhost:8545"
```

Comment:
Not sure what is best practice of config in elixir now. Previous idea was to run `MIX_ENV=<env> mix blabla` for each env so we don't need to re-config the common things. However, seems like elixir is moving away from config system?

Probably we can start to have some `local_docker.env` file for local docker testings and cleanup the other configs?

Anyway, not a blocker of this PR. But I am curious to learn from elixir expert what should be a good solution 😛